migrations/V5__add_cenotaph_to_runes.sql

```diff
@@ -0,0 +1,3 @@
+alter table runes add column cenotaph BOOLEAN;
```

Comment:
Since we don't have an official release out yet, I think we should add this to to original `runes` table migration